### PR TITLE
Move ping function to moco-agent.

### PR DESCRIPTION
**Pull request by kfyharukz**
Signed-off-by: Kazuhito MATSUDA <kazuhito.matuda@gmail.com>

### CODE REVIEW

File: cmd/moco-agent/cmd/ping.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+)
+
+var pingCmd = &cobra.Command{
+	Use:   "ping",
+	Short: "Send ping to a MySQL instance",
+	Long:  `Send ping to a MySQL instance.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		_, err := os.Stat("/var/lib/mysql/init-once-completed")
+		if err != nil {
+			return nil
```
Comment (zoetrope) [inline cmd/moco-agent/cmd/ping.go:22]: I think it should return error code 1.

**Review by zoetrope — CHANGES_REQUESTED** [review]

File: Makefile [hunk 2]
```diff
@@ -33,7 +35,7 @@ test:
 	docker run -v $(PWD):/go/src/github.com/cybozu-go/moco -e GOPATH=/tmp --rm mysql-with-go:latest sh -c "CGO_ENABLED=0 go test -v ./initialize"
 
 # Build entrypoint binary
```
Comment (masa213f) [inline Makefile:37]:
```suggestion
# Build moco-agent binary
```

File: cmd/moco-agent/cmd/ping.go [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+const (
+	credentialConfPathFlag = "credential-conf-path"
+)
+
+var pingCmd = &cobra.Command{
+	Use:   "ping",
+	Short: "Send ping to a MySQL instance",
+	Long:  `Send ping to a MySQL instance.`,
+	RunE: func(cmd *cobra.Command, args []string) error {
+		_, err := os.Stat("/var/lib/mysql/init-once-completed")
```
Comment (masa213f) [inline cmd/moco-agent/cmd/ping.go:20]: https://github.com/cybozu-go/moco-agent/blob/bebff9ddd80b425da6a5df75575f53fb2ac27775/initialize/init_test.go#L19
Can we use this variable?

File: Makefile (hunk 2 above)
Comment (kfyharukz) [reply Makefile:37]: 🙇

**Review by masa213f — APPROVED** [review]
LGTM

**Review by zoetrope — APPROVED** [review]
LGTM

**Pull request merged**